packages/redis/src/index.js

```diff
@@ -12,11 +12,13 @@ class KeyvRedis extends EventEmitter {
       this.redis = uri
     } else {
       options = Object.assign(
-        {},
+        {
+          emitErrors: true
+        },
         typeof uri === 'string' ? { uri } : uri,
         options
       )
-      this.redis = new Redis(options.uri, options)
+      this.redis = new Redis(options.uri, options.redisOptions)
```

Comment:
hum `redisOptions` is not documented.

Reply:
BTW what do you think to simply pass options to avoid `redisOptions` level of indentation?

Something like this:

```js
new Keyv({
  store: new KeyvRedis({
    uri: "redis://user:pass@localhost:6379",
     db: 1
  }),
});
```

Reply:
That is what was being done before but it was confusing for me to figure out which options goto redis and which goto the KeyvRedis instance, typescript also didn't like it, but if its needed we can revert it to options merging and make KeyvRedis.Options extend RedisOptions

Reply:
That right but the thing is we don't necessarily need to know what options `ioredis` needs. We just pass `options` and the `new Redis` constructor will pick up from there

Reply:
Alright sounds good, should I remove mongoOptions from mongo as well then ?

Reply:
I think so, sorry for breaking things 😬